### PR TITLE
feat(docs): add openapi docs

### PR DESCRIPTION
**Pull request by rudyardrichter**
Add openapi docs for the arborist API.

### CODE REVIEW

**Comment by rudyardrichter** [comment]
Use this link: https://raw.githubusercontent.com/uc-cdis/arborist/feat/docs/docs/openapi.yaml
in the swagger UI: http://petstore.swagger.io/
to look at the rendered documentation

File: docs/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,760 @@
...
+    get:
+      tags:
+        - role
+      description: List all the roles registered in arborist.
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              # example JSON:
```
Comment (Avantol13) [inline docs/openapi.yaml]: dead code, could remove unless there's a valid reason to keep it.

File: docs/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,760 @@
...
+              description: an explanation of the error that occurred
+            code:
+              type: integer
+              description: the HTTP error code
+      example:
+        error:
+          message: "resource with path `/foo/bar` does not exist"
+          code: 404
+    Resource:
+      # example JSON:
```
Comment (Avantol13) [inline docs/openapi.yaml]: dead code?

File: docs/openapi.yaml (hunk 1 above)
Comment (rudyardrichter) [reply docs/openapi.yaml]: will remove

File: docs/openapi.yaml (hunk 2 above)
Comment (rudyardrichter) [reply docs/openapi.yaml]: also removed

**Pull request merged**